src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -178,14 +178,15 @@ private function getPropertyNameCollectionFactoryContext(ResourceMetadata $resou
     private function getHydraProperties(string $resourceClass, ResourceMetadata $resourceMetadata, string $shortName, string $prefixedShortName): array
     {
         $properties = [];
-        foreach ($this->propertyNameCollectionFactory->create($resourceClass, $this->getPropertyNameCollectionFactoryContext($resourceMetadata)) as $propertyName) {
+        $context = $this->getPropertyNameCollectionFactoryContext($resourceMetadata);
```

Comment:
> Sorry but it still doesn't look good to me. The context must be propagated from normalize and denormalize, not re-created.

Are you talking about this line?

May you help me a bit or edit my PR? I don't know how to manage the Hydra part for that. :-|